test/unit/frontend/github-setup.test.ts

```diff
@@ -22,27 +22,53 @@ describe("Github Setup", () => {
 	});
 
 	describe("#GET", () => {
-		it("should return HTML when jiraHost is missing", async () =>
+		beforeEach(async () => {
+			await Installation.create({
+				jiraHost,
+				clientKey: "abc123",
+				secrets: "def234",
+				sharedSecret: "ghi345",
+			});
+		});
+
+		it("should return redirect to github oauth flow for GET request if token is missing", async () =>
```

Comment:
I had to update the tests for 'GET' as we are now passing `oauth.checkGithubAuth` to the github/setup endpoint for get requests. @mboudreau would love your guidance here. Is there anything else I should be testing? Do I need to change these/

Reply:
Phew!